app/views/shared-parental-leave-planner/planner-birth.html

```diff
@@ -160,9 +166,13 @@ <h2 class="govuk-heading-m">
         ]
       }) }}
       <div class="pay-only">
+      <h2 class="govuk-heading-m">
+        Statutory pay amounts
+      </h2>
+        <p>

```

Comment:
Indentation here is slightly off. The `<p>` should be on the same level as the `<h2>`